polars/polars-lazy/src/tests/functions.rs

```diff
@@ -0,0 +1,24 @@
+use super::*;
+
+#[test]
+#[cfg(feature = "format_str")]
```

Comment:
Can you move this tests to the integration tests? I want to keep polars-lazy slim on tests.

Reply:
Sure can, I anticipated you'd have somewhere better to put the test, lol.